wherobots/db/connection.py

```diff
@@ -134,6 +137,9 @@ def __listen(self) -> None:
                 # On a state_updated event telling us the query succeeded,
                 # ask for results.
                 if kind == EventKind.STATE_UPDATED:
+                    logging.info(
```

Comment:
Do you want to keep that?

Reply:
Yes and no. Since result_uri and size is currently part of the state updated event (and not the retrieve result) we might want to have a way to surface that info programatically. Right now it's an easy way to just log dump it, but honestly it's a poor way to surface it.

We should talk about really making it part of a request result call even if it's another extra round or have a nicer way to handle it from the client side.